src/vs/workbench/contrib/splash/browser/partsSplash.ts

```diff
@@ -110,8 +110,6 @@ export class PartsSplash {
 
 		// remove initial colors
 		const defaultStyles = mainWindow.document.head.getElementsByClassName('initialShellColors');
-		if (defaultStyles.length) {
-			mainWindow.document.head.removeChild(defaultStyles[0]);
-		}
+		defaultStyles?.[0].remove();
```

Comment:
@remcohaszing this is not a good change, what if the array is empty?

Reply:
This was indeed wrong. Fixed it in #214348.